### DISCUSSION TITLE
Is there any idea about getting a list localized from a file like plist, txt?

### DISCUSSION THREAD

**Discussion by vanle57** [SwiftGen/SwiftGen#1000] [Q&A]
I would like to make a plist file to read a list of names and localize those strings. With the system's localized function, I can define a localized key as its values and then read a file and use `"localized_key".localized` but in SwiftGen, I have no idea. I just don't want to hard code a list.

**Comment by djbe** [comment]
Could you provide an example of the plist file, and what code you'd need? 

I'm pretty sure you can use the `plist` parser and the built-in template to create your own custom template that achieves what you want.
